src/__tests__/helpers.js

```diff
@@ -0,0 +1,166 @@
+import {
+  guessMaxImageWidth,
+  bytesToSize,
+  selectSrc,
+  fallbackParams,
+} from '../components/helpers'
+
+describe('guessMaxImageWidth', () => {
+  it('Should calculate the maximum image width', () => {
+    const dimensions = {
+      width: 400,
+      height: 100,
+    }
+    const expected = dimensions.width
+    const result = guessMaxImageWidth(dimensions)
+    expect(result).toEqual(expected)
+  })
+
+  it('Should calculate the maximum image width with screen changes', () => {
+    Object.defineProperty(window.screen, 'width', {
+      get: () => {
+        return 100
+      },
+    })
+    window.innerWidth = 50
+    const dimensions = {
+      width: 400,
+      height: 100,
+    }
+    const expected =
+      (dimensions.width / window.innerWidth) * window.screen.width
+    const result = guessMaxImageWidth(dimensions)
+    expect(result).toEqual(expected)
+  })
+
+  it('Should calculate the maximum image width with screen changes and scroll', () => {
+    // window.screen.width defined in test above.
```

Comment:
this is not the best idea. If you want to provide information to more then one test use `beforeEach`

Reply:
Good point. I've fixed this to make it more clear.